Dockerfile

```diff
@@ -1,18 +1,13 @@
-FROM golang:1.15.4-alpine3.12 as build
-
-COPY ./ /go/src/github.com/chatwork/sendgrid-stats-exporter
-WORKDIR /go/src/github.com/chatwork/sendgrid-stats-exporter
-
-RUN go mod download \
-#    && go test ./... \
-    && CGO_ENABLED=0 GOOS=linux go build -o /bin/exporter
-
-FROM alpine:3.12
-
-RUN apk --no-cache add ca-certificates \
-     && addgroup exporter \
-     && adduser -S -G exporter exporter
-USER exporter
-COPY --from=build /bin/exporter /bin/exporter
-
-ENTRYPOINT [ "/bin/exporter" ]
\ No newline at end of file
+FROM golang:1.17.6-bullseye as builder
+WORKDIR /app
+COPY go.* ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -o exporter
+
+FROM debian:bullseye-slim
```

Comment:
Why did you choose debian image instead of other smaller base images?

Reply:
This was not a good repository that I referred to. I'll make it smaller.

Reply:
See https://github.com/ubie-oss/sendgrid-stats-exporter/pull/1/commits/2f123984ad42e943074c1a37f1a4efabc24f70c6 https://github.com/ubie-oss/sendgrid-stats-exporter/pull/1/commits/72bcaca336aaa30813940daf173339c6a4f1c472